modules/mcg-about-cluster-sizing.adoc

```diff
@@ -0,0 +1,36 @@
+
+:_content-type: CONCEPT
+:imagesdir: ../../images
+
+[id="about-openshift-cluster-sizing-mcg"]
+= About OpenShift cluster sizing for the Multicloud GitOps Pattern
+
+To understand cluster sizing requirements for the Multicloud GitOps pattern, consider the following components that are the Multicloud GitOps pattern deploys on the datacenter or the hub OpenShift cluster:
```

Comment:
This sentence reads a little oddly. Might need to remove the "are" from "consider the following components that are the"